src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/desktop/SortingFilterDesktop.java

```diff
@@ -0,0 +1,40 @@
+package com.wikia.webdriver.elements.mercury.components.discussions.desktop;
+
+import com.wikia.webdriver.common.logging.PageObjectLogging;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.wikia.webdriver.common.logging.PageObjectLogging'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)